vm/array.go

```diff
@@ -990,6 +944,34 @@ func (a *ArrayObject) toJSON() string {
 	return out.String()
 }
 
+// Retrieves an object in an array using Integer index; common to `[]` and `at()`.
+func (a *ArrayObject) index(t *thread, args []Object, blockFrame *callFrame) Object {
```

Comment:
Why passing blockFrame? Seems that it's not required for this method.

Reply:
Right! Removed.